src/debugger/ios/deviceRunner.ts

```diff
@@ -149,14 +149,14 @@ export class DeviceRunner {
         this.cleanup();
 
         return this.mountDeveloperImage().then(function(): Q.Promise<any> {
-            let result = new Node.ChildProcess().spawnWaitUntilStarted("idevicedebugserverproxy",  [proxyPort.toString()]);
-            return result.outcome.then(() => this.nativeDebuggerProxyInstance = result.spawnedProcess);
+            let result = new Node.ChildProcess().spawn("idevicedebugserverproxy",  [proxyPort.toString()]);

```

Comment:
Should `new Node.ChildProcess()` be in an instance variable to be consistent with the other files in this PR?